scripts/components/RecentChanges/ChangeRow.js

```diff
@@ -24,9 +29,46 @@ export default function ChangeRow({
 }) {
   const classes = useStyles()
   const theme = useTheme()
-  const matches = useMediaQuery(theme.breakpoints.up('sm'))
+  //TODO - Restore the "useMediaQuery" and fix long names
+  const matches = true
 
   return (
+    <Box display="flex" alignItems="center">
+      <Box padding="0.5rem">
+        {eventIcon}
+      </Box>
+      <Box padding="0.25rem">
+        {
+          (eventName === 'CREATE') ? (
+            <span className={classes.create}>{`${titleCase(eventName)}d`}</span>
+          ) : (eventName === 'UPDATE') ? (
+            <span className={classes.rename}>{`${titleCase(eventName)}d`}</span>
+          ) : <span>{`${titleCase(eventName)}d`}</span>
+        }
```

Comment:
Can replace this with changing just the class using either plain logic or looking at clsx

Reply:
My idea is to just replace the hardcoded rules with something consistent by using another flexbox or something, it's on my TODO